diff --git a/node_modules/native-keymap/binding.gyp b/node_modules/native-keymap/binding.gyp
index 1111111..2222222 100644
--- a/node_modules/native-keymap/binding.gyp
+++ b/node_modules/native-keymap/binding.gyp
@@ -16,6 +16,7 @@
           'AdditionalOptions': [
             '/guard:cf',
             '/sdl',
+            '/wd4996',
             '/W3',
             '/we4146',
             '/we4244',
